### DISCUSSION TITLE
trigger_state is error when restart application

### DISCUSSION THREAD

**Discussion by Gavinluo** [quartznet/quartznet#1525] [Q&A]
my app host in iis , when i  force restart iis , trigger_state will be **ERROR**  and not fire , only change it to **WAITING** , the tigger is work , is this a bug? 

**Version used**

version: 2.5.0.0

**To Reproduce**

 happened occasionally

**Expected behavior**

quartz will auto repair this error states

**Comment by lahma** [comment]
Try upgrading to latest version. 2.5 is over 5 year old version.

**Reply by Gavinluo** [reply]
hi lahma , thanks for your reply, i will upgrading to lasest version and check it is will reappear
